File: mcp_servers/js/clients/src/client_and_server_config.ts
export const ClientsConfig:any = [
    "MCP_CLIENT_OPENAI",
    "MCP_CLIENT_AZURE_AI",
    "MCP_CLIENT_GEMINI",
    // "CLAUDE",
]

export const ServersConfig:any = [
    {
        server_name :"WORDPRESS",
        server_features_and_capability:`WORDPRESS`,
        path : "build/index.js"
    },
    {
        server_name :"AIRTABLE-MCP",
        server_features_and_capability:"Airtable integration via MCP",
        path : "build/index.js"
    }
]

File: mcp_servers/python/clients/src/client_and_server_config.py
ClientsConfig =[
    "MCP_CLIENT_AZURE_AI",
    "MCP_CLIENT_OPENAI",
	"MCP_CLIENT_GEMINI"
]
ServersConfig = [
	# {
	# 	"server_name": "MCP-GSUITE",
	# 	"command":"uv",
	# 	"args": [
	# 		"--directory",
	# 		"../servers/MCP-GSUITE/mcp-gsuite",
	# 		"run",p
	# 		"mcp-gsuite"
	# 	]
	# },
	 {
        "server_name": "AIRTABLE",
        "command": "python",
        "args": [
            "../../../../../servers/AIRTABLE/airtable_mcp/src/server.py",
            "--token", "your_api_key_here",
            "--base", "appToc26GLbyyEF1e",
            "--dev"
        ]
    }
]